### DISCUSSION TITLE
[SOLVED] Cuando se utiliza un external survey (compartiendo el link), no sé visualiza correctamente el formulario

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#101] [4. Bug Reports] [answered]
Hola

En una implementación que estoy haciendo actualmente. Me percaté que, en el momento de usar el link publico, el formulario sale cortado (o ajustado a lo que se pueda ver). Esto quiere decir que para formularios con pocos campos, no existe inconveniente. Sin embargo cuando el fomrulario es extenso, sólo se ve una parte. Para ver el resto, se debe disminuir la ventana (50%), para que se vea completamente.

Dejo las imagenes adjuntas:

Cuando está la 100%
![image](https://user-images.githubusercontent.com/61419471/115604734-b552a180-a2af-11eb-9af3-db5dd5667e3b.png)

Cuando está al 50%
![image](https://user-images.githubusercontent.com/61419471/115604749-ba175580-a2af-11eb-878c-1b04424e2352.png)

Quedo atento si ya está considerado o lo debo reportar internamente.

Gracias

**Comment by agallardol** [comment, marked as answer]
@NicolasPLG En la última versión de Cotalker, public surveys no funciona de forma responsive y tiene algunos problemas para mostrar formularios muy extensos. Con las últimas mejoras que hicimos esto ya no debería ocurrir. El siguiente release que hagamos debería incluirlas!

**Reply by NicolasPLG** [reply]
Gracias @agallardol por la información, esperaré el release
